tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml

```diff
@@ -130,14 +133,14 @@ jobs:
     displayName: 'Download Pipeline Artifact - Win x64'
     inputs:
       buildType: 'current'
-      artifactName: 'drop-onnxruntime-java-win-gpu-x64'
+      artifactName: 'onnxruntime-java-win-x64-cuda'
```

Comment:
Please keep the prefix unchanged. The cpu one is still "drop-onnxruntime-java-win-x64". If you don't change that, it's inconsistent. But if you change that, you will need to change a lot of places in the c-api-cpu.yml. Given we only have 3 days left,  I think it's not a good time to do this.

Reply:
I updated the Artifact names that are related to "GPU" and I kept cpu one untouched.

Reply:
Following three Artifacts are the final results of combined GPU package:
onnxruntime-linux-x64-gpu
onnxruntime-win-x64-gpu
onnxruntime-java-gpu

Reply:
i think there is something wrong with onnxruntime-win-x64-gpu zip
the copy zip file step finds 0 files. 

Starting: Copy zip file to: D:\a\_work\1\a
==============================================================================
Task         : Copy files
Description  : Copy files from a source folder to a target folder using patterns matching file paths (not folder paths)
Version      : 2.190.1
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/copy-files
==============================================================================
found 0 files
Finishing: Copy zip file to: D:\a\_work\1\

Reply:
Like I said, it is inconsistent.  The GPU part of jar packaging looks for "onnxruntime-java-win-x64-cuda", but CPU looks for "**drop**-onnxruntime-java-win-x64". And you'd need to modify the scripts to be compatible with both.